teachers_digital_platform/css/organisms/expandable.less

```diff
@@ -0,0 +1,7 @@
+// Expandable header
+
+.o-expandable_header-left {
+    .tdp-activity-search & {
+        width: 65%;
```

Comment:
It might also be nice to include a comment about the reasoning for choosing this number.